src/additional_zones.rs

```diff
@@ -48,6 +49,8 @@ pub fn compute_additional_cities(zones: &mut Vec<Zone>, pbf_path: &str) {
         candidate_parent_zones
             .into_iter() //TODO into_par_iter
```

Comment:
you can remove this TODO now :stuck_out_tongue_winking_eye:

Reply:
But... I like... :cry: